EvidenceApi/V1/Gateways/NotifyGateway.cs

```diff
@@ -67,6 +83,11 @@ private NotificationResponse Deliver(DeliveryMethod deliveryMethod, string templ
             };
         }
 
+        private NotificationResponse DeliverEmail(string templateId, EvidenceRequest evidenceRequest, Dictionary<string, object> personalisation)
```

Comment:
We could make this method more generic by changing the signature to `DeliverEmail(string templateId, string emailAddress, Dictionary<string, object> personalisation)`

Reply:
I did wonder we could refactor `Deliver` to `Deliver(DeliveryMethod deliveryMethod, string templateId, string emailAddress, string mobileNumber, Dictionary<string, object> personalisation)`, but wasn't sure if we wanted to pass in a null mobileNumber

Reply:
Yes, I think it's better to not have null arguments if possible